go/vt/vttablet/tabletserver/vstreamer/testenv/testenv.go

```diff
@@ -41,7 +42,12 @@ import (
 	vttestpb "vitess.io/vitess/go/vt/proto/vttest"
 )
 
-const DBName = "vttest"
+const (
+	DBName               = "vttest"
+	DefaultCollationName = "utf8mb4_0900_ai_ci"
+)
+
+var DefaultCollationID = collations.MySQL8().LookupByName(DefaultCollationName)
```

Comment:
Why not `collations.MySQL8().DefaultConnectionCharset()`? That returns the exact same thing. You can then also lookup the name for that. I think that's more semantically correct and avoids duplicating the `utf8mb4_0900_ai_ci` as a default value.

Reply:
I did think about that as well and started off using it. I may go back to it -- don't feel strongly at all. It's an inconsequential part of this PR IMO and I'm fine either way.